pandas/core/aggregation.py

```diff
@@ -196,3 +266,49 @@ def maybe_mangle_lambdas(agg_spec: Any) -> Any:
         mangled_aggspec = _managle_lambda_list(agg_spec)
 
     return mangled_aggspec
+
+
+def _relabel_result(
+    result: Union[ABCSeries, ABCDataFrame],
+    func: Dict[str, List[Union[Callable, str]]],
+    columns: Tuple,
+    order: List[int],
+) -> Dict[str, ABCSeries]:
```

Comment:
Can you replace ABCSeries with Series? The former is created at runtime so doesn't help with static analysis

Reply:
is this possible?

Reply:
changed